src/style.css

```
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: "Avenir-Book";
+  font-weight: 300;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  font-weight: bold;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
```

```suggestion
@font-face {
  font-family: "Avenir";
  font-weight: 300;
  src: url("fonts/Avenir-Book.ttf");
}

@font-face {
  font-family: "Avenir";
  font-weight: bold;
  src: url("fonts/Avenir-Heavy.ttf");
}
```

You can use one font-family but with differet font-weights